package.json

```diff
@@ -44,12 +49,16 @@
     ]
   },
   "dependencies": {
-    "ember-cli-babel": "^7.23.0",
+    "ember-cli-babel": "^7.26.5",
+    "ember-cli-htmlbars": "^5.7.1",
```

Comment:
We don't have any `addon/**/*.hbs` files, mind moving this back down into devDeps?